src/pages/Join/Join.js

```diff
@@ -3,7 +3,133 @@ import './Join.scss';
 
 class Join extends Component {
   render() {
-    return <div></div>;
+    return (
+      <main className="Join">
+        <div className="container">
+          <h1>Sign Up</h1>
+          <ul className="join-inner">
+            <li>
+              <label>ID</label>
+              <input type="text" name="id" />
+              <span>아이디를 입력해 주세요.</span>
+            </li>
+            <li>
+              <label>Password</label>
+              <input type="password" name="id" maxlength="16" />
+              <span>
+                (영문 대소문자/숫자/특수문자 중 2가지 이상 조합, 8자~16자)
+              </span>
+            </li>
+            <li>
+              <label>Password Check</label>
+              <input type="password" name="id" maxlength="16" />
+              <span></span>
+            </li>
+            <li>
+              <label>Name</label>
+              <input type="text" name="id" />
+              <span></span>
+            </li>
+
+            <li className="mobile">
+              <label>Mobile</label>
+              <select className="mobile1">
+                <option>010</option>
+                <option>011</option>
+                <option>016</option>
+                <option>017</option>
+                <option>018</option>
+              </select>
+              <input type="text" maxlength="4"></input>
+              <input type="text" maxlength="4"></input>
+              <span></span>
```

Comment:
```suggestion
              <input type="text" maxlength="4" />
              <input type="text" maxlength="4" />
              <span />
```
- 셀프 클로징 태그 써주세요
- 빈 span 태그는 뭔가요?